support-utils/cre_astro_software_gather_logs.sh

```diff
@@ -0,0 +1,392 @@
+#!/bin/bash
+timestamp=`date '+%d/%m/%Y %H:%M:%S'`
+# Put the name of the astronomer ASTRONOMER_NAMESPACE below
+#export ASTRONOMER_NAMESPACE=astronomer
+#export DIR="/tmp"
+#export BASEDOMAIN=nandlal51.astro-cre.com 
```

Comment:
This comment data should be replaced with example.com or the comment should be removed.

https://www.rfc-editor.org/rfc/rfc2606

Reply:
used example.com to fix this